src/prr.rs

```diff
@@ -26,6 +47,12 @@ struct PrrConfig {
     url: Option<String>,
 }
 
+#[derive(Debug, Deserialize)]
+struct PrrLocalConfig {
```

Comment:
Recommend we have a section name for the local config. So like how main config file has a `[prr]` section, maybe local config has a `[override]` section? For consistency.

Reply:
'Cause `override` is a Rust-keyword, and we should always refer with `r#override` i used `local`, ok?

Reply:
Yeah, local is good